modules/apps/batch-planner/batch-planner-test/src/testFunctional/macros/ImportExport.macro

```diff
@@ -446,6 +477,26 @@ definition {
 		WaitForVisible(locator1 = "Button#PLUS");
 	}
 
+	@summary = "Default summary"
+	macro rewriteDownloadedExportJSONLFile(fileName = null, originalParameter = null, newParameter = null) {
```

Comment:
because the JSON and JSONL have different content on the file, so now it only can rewrite JSONL file, if we want to rewrite JSON file, I think we need to modify the macro

Reply:
but you parametrise the content right?

Reply:
actually, I copy the full original file content and write new file content with original content and new content

Reply:
@Hijky Can you explain what is original content and what is new content? Just for me to understand the purpose of it, thanks!

Reply:
@magjed4289 
the original content == the export student entry
the new content == the export student entry + a new student entry
and with this method, we don't need to create a new file, we only need to rewrite the export file in the temp folder 'poshi'
cc @yanan

Reply:
that a nice programming idea, isn't it faster though to have another file wuth created student entry and the new one together? 🤔

Reply:
but I don't find a method can add new content in an existing file, the write method will reword all the file content

Reply:
why using an existing file? why not creating a file that would have an information of the created entry and new entry and put it in the repo? Do we need any parameters that would change with different environment/siteId?

Reply:
two parameters we need to change for this test 'ERC' 'name', I will try to create a file in our repo to replace to use this method

Reply:
@Hijky parameters ERC and name are totally controllable by us and we can set them in as precondition in a file. I think there's no need to create a mechanism to merge file, we can use one that we create beforehand, as we did in manual tests.

Reply:
@magjed4289 sure, I have modified and added a JSONL file in the repo, could you review it again? Thanks!